repos/fdbt-site/src/pages/managePurchaseMethod.tsx

```diff
@@ -73,13 +86,20 @@ const ManagePurchaseMethod = ({ inputs, csrfToken, errors, editMode }: ManagePur
                 <div className="govuk-grid-column-two-thirds">
                     <CsrfForm action="/api/managePurchaseMethod" method="post" csrfToken={csrfToken}>
                         <>
-                            <h1 className="govuk-heading-l">Provide purchase method details</h1>
-                            <span id="service-list-hint" className="govuk-hint">
+                            <h1 className="govuk-heading-l">
+                                Provide {isCapped ? 'capped' : ''} purchase method details
+                            </h1>
+                            {isCapped ? (
+                                <span id="purchase-method-hint" className="govuk-hint">
+                                    Capped purchase methods are for capped products
```

Comment:
```suggestion
                                    Capped purchase methods have fewer options than an ordinary purchase method, because they are the specific ways in which a passenger can purchase one of your capped products.
```